File: Strings/pagina06/3300/3300.cpp
#include <iostream>
using namespace std;

int main(){

    string n;
    cin >> n;

    if (n.find("13") != string::npos){
        cout << n << " es de Mala Suerte" << endl;
    }
    else{ cout << n << " NO es de Mala Suerte" << endl; }

    return 0;
}

File: Strings/pagina06/3300/3300.java
import java.util.Scanner;
import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        Scanner in = new Scanner(system.in);

        String line = in.nextLine();
        String response = "";

        if(line.contains("13")){
            response = line + " es de Mala Suerte";
        } else {
            response = line + " NO es de Mala Suerte";
        }

        System.out.println(response);

        in.close();
    }
}